designs/containers/workload-discovery/workload-discovery.md

```diff
@@ -30,14 +30,10 @@ To create a traffic redirection using Nuage for Kubernetes the only required inf
 > Note: The assumptions below refer to design changes not yet fully described in this document but necessary to understand some of its current content. In the next revision of this document those details will be added (look for **TBD on the next revision** along this document) and these assumptions removed.  
 
 * **Security Group:**
-The security group entity and DTO in OSC will have a set of labels. These labels will be strings with the format `key=value`. For the first release of this feature all the `key` values of the labels in a given security group will always be the same, i.e.: security=webserver, security=backend, etc.
+For the first release of this feature all the `key` values of the labels in a given security group will always be the same, i.e.: security=webserver, security=backend, etc.
 
 * **Pod Security Group Member:**
-A new type of security group member is created, **Pod** along with the type **PodPort**. The **Pod** will store a `namespace`, `name`, `externalid`, `nodename` and a collection of `PodPort`.  The **PodPort** will store an `externalid`, a `collection of macaddresses` and a collection of `ipaddress`. 
-> Note: Although currently a pod can only have one ip address [support for multiple ip addresses and networks is intended](#k8s-pods-with-multiple-ips-issue).
-
-* **Virtualization Connector:**
-A new type of Virtualization Connector (VC) will be added: **Kubernetes**. 
+A new type of security group member is created, **Pod**. A pod can be part of many security groups at the same time.  

```

Comment:
We should clarify what this means if you have a pod part of multiple security groups, then how do we handle the multi policy problem?

We can add that even though it can be part of multiple SG's, it is only active in one security group. either because its marked for deletion in the other security group or that its part of a dynamic member set and will be ignored/failed as part of the sync